lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1693,8 +1693,6 @@ void PrimUpdaterManager::onProxyContentChanged(
             }
         }
     }
-
-    executeAdditionalCommands(context);
 }
```

Comment:
The auto-edit function above calls editAsMaya which already takes care of calling executeAdditionalCommands. Furthermore, the removed call was executed outside the undo/redo muting, and was "leaking" undo items.